src/Eccube/Resource/template/install/migration_frame.twig

```diff
@@ -69,6 +65,7 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
     ================================================== -->
 <!-- Placed at the end of the document so the pages load faster -->
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script>window.jQuery || document.write('<script src="{{ publicPath }}template/install/assets/js/jquery-1.11.1.min.js?v={{ constant('Eccube\\Common\\Constant::VERSION') }}"><\/script>')</script>
```

Comment:
use the same directory structure (front / admin)
like this :  `html/template/default/js/vendor/jquery-1.11.3.min.js`